webapp/packages/core-theming/src/styles/_variables.scss

```diff
@@ -50,3 +50,9 @@ $app-height: 100vh !default;
   --theme-group-element-radius: #{$theme-group-element-radius};
   --theme-menu-bar-small-action-radius: #{$theme-menu-bar-small-action-radius};
 }
+
+@layer components {
+  body {
+    --dbv-kit-control-outline-color: var(--theme-primary);
+  }
+}
```

Comment:
please also remove scss styles related to buttons (`_button.scss`, maybe some else)

Reply:
we can't do it right now, a few components extend those styles

Reply:
ok